### DISCUSSION TITLE
Support for WebVR

### DISCUSSION THREAD

**Discussion by parasyte** [melonjs/melonJS#1029] [Ideas]
Yep, apparently this is a thing.

http://blog.bitops.com/blog/2014/06/26/first-steps-for-vr-on-the-web/

IMHO, the VR devices are still too young to be completely immersive. And apart from some crazy cool tech demos, it's really only suitable for first person shooters and simulators (weird). Anyway, we should investigate opportunities!

**Comment by obiot** [comment]
yeah, can't wait to see our platformer example in VR :P

**Comment by obiot** [comment]
In case you have not seen that article yet:
https://hacks.mozilla.org/2015/09/stereoscopic-rendering-in-webvr/

**Comment by agmcleod** [comment]
I think having the engine support 3d is a bit of a blocker before this ;)

**Comment by parasyte** [comment]
TBH, we have Vector3D in place now, and we're just a few steps away from passing the z-coordinate to the vertex shaders. ;) That makes it 3D.
